docs/docs/rest-api/public/api/v2/types/pod-instance-status.raml

```diff
@@ -0,0 +1,66 @@
+#%RAML 1.0 DataType
+type: object
+(scalaType): PodInstanceStatus
+additionalProperties: false
+properties:
+  id:
+    type: !include name.raml
+    description: |
+      Unique ID of this pod instance in the cluster.
+      TODO(jdef) Probably represents the Mesos executor ID.
+  status:
+    type: string
+    (scalaType): PodInstanceState
+    enum: [ PENDING, STAGING, STABLE, DEGRADED, TERMINAL ]
```

Comment:
unclear how/why this was promoted to an enum again. need to sync w/ @jasongilanfarr